jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml

```diff
@@ -403,6 +407,12 @@
       <artifactId>guava</artifactId>
     </dependency>
 
+    <!-- KIE Workbench Playground Repository -->
+    <dependency>
+      <groupId>org.kie.workbench.playground</groupId>
```

Comment:
Please make sure the example has proper &lt;description&gt; in the pom as it gets used in the UI as mentioned in the [following issue](https://github.com/kiegroup/kie-wb-playground/issues/9)